joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

first_wheel_controller:
  type: velocity_controllers/JointVelocityController
  joint: first_wheel_joint
  publish_rate: 50
  pid: {p: 10.0, i: 1.0, d: 0.0}

second_wheel_controller:
  type: position_controllers/JointPositionController  # Change to position control
  joint: second_wheel_joint
  publish_rate: 50
  pid: {p: 0.1, i: 0.01, d: 0.0}

third_wheel_controller:
  type: effort_controllers/JointEffortController  # Change to torque (effort) control
  joint: third_wheel_joint
  publish_rate: 50